examples/nirfmxbluetooth/acp-basic.py

```diff
@@ -322,18 +321,15 @@ def check_for_warning(response, instrument):
             print(f"Upper Margin (dB)                    : {upper_margin[i]}\n")
 except grpc.RpcError as rpc_error:
     error_message = rpc_error.details()
+    for key, value in rpc_error.trailing_metadata() or []:
```

Comment:
The build process runs type checking on a few examples (@reckenro why not all?) which necessitated this change that you should also use:
```
for metadatum in rpc_error.trailing_metadata() or []:
    if metadatum.key == "ni-error":
        error_message += f"\nError status: {metadatum.value.decode('utf-8')}"
```

Reply:
Got it, updated

Reply:
Yeah, I honestly forget the historical reason. If you look at the [validate_python.yml](https://github.com/ni/grpc-device/blob/main/.github/workflows/validate_python.yml) you can see it does the styleguide lint check on all examples. But it only runs validate_examples.py (which does the Mypy type checks among other things) on rfmx and xnet example folders.

The reason might've been issues with betterproto examples causing issues with mypy and/or us just focused on rfmx and xnet at the time and weren't expecting to change the others anytime soon so we didn't bother with them. I can look into it and see about making a Task/Bug/Tech Debt to sort it out (probably after these example updates go in).

Reply:
The reason that it doesn't run on all examples is because the examples from before `validate_examples` was added weren't `mypy` compliant (most commonly reusing `response` variables) or had pypi deps beyond what the `validate_examples` script adds (including-but-not-limited-to `betterproto`). We just didn't go back and fix them, since it's most useful when examples are under development and changing to examples is kind of fussy and requires testing  against a server with simulated HW.

I think it'd be great to get all of the `validate_examples` running, especially since we're having to touch all of the examples for this release anyway. That's also helpful for running the examples and even testing some error cases (invalid resource). Currently that's only a convenience for manual testing, though potentially that could run on the HW CI (@DavidCurtiss ).